src/app/_services/api.service.ts

```diff
@@ -160,15 +179,16 @@ export class ApiService {
   }
 
   public handleError(error: HttpErrorResponse) {
+    if (!!error.error.result.message) { this.alert.error(error.error.result.message); }
     if (error.error instanceof ErrorEvent) {
       // A client-side or network error occurred. Handle it accordingly.
-      console.error('An error occurred:', error.error.message);
+      console.error('An error occurred:', error.error);
```

Comment:
should use `this.debug.log`. You can set the type to be 'error' ex: `this.debug.log('error', error.error);`